### PR TITLE
Remove setting default value to 1000 and rename uploadcache security …

### PR DESCRIPTION
**Pull request by mphammer**
…context map-key to match the pod name

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1173#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/apps/blackduck/latest/containers/postgres.go [hunk 1]
```diff
@@ -81,16 +80,10 @@ func (c *Creater) GetPostgres() *postgres.Postgres {
...
-		postgresSecurityContext := apputil.GetSecurityContext(c.blackDuck.Spec.SecurityContexts, "blackduck-postgres")
-		if postgresSecurityContext == nil {
-			postgresSecurityContext = &api.SecurityContext{
-				FsGroup:    util.IntToInt64(26),
-				RunAsUser:  util.IntToInt64(26),
-				RunAsGroup: util.IntToInt64(26),
-			}
+		if postgresSecurityContext := apputil.GetSecurityContext(c.blackDuck.Spec.SecurityContexts, "blackduck-postgres"); postgresSecurityContext != nil {
+			utilPostgresSecurityContext := util.SecurityContext(*postgresSecurityContext)
+			postgresConfig.SecurityContext = &utilPostgresSecurityContext
```
Comment (yashbhutwala) [inline pkg/apps/blackduck/latest/containers/postgres.go]: Just wondering why this can't be made like the other pods now?

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1173#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1173#pullrequestreview-341334708" title="Approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/apps/blackduck/latest/containers/postgres.go (hunk 1 above)
Comment (mphammer) [reply pkg/apps/blackduck/latest/containers/postgres.go]: Uses the same logic now :)

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1173#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1173#issuecomment-573663808" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.12.x/OWNERS)~~ [mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@yashbhutwala: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1173#issuecomment-573663886):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@yashbhutwala: new pull request created: #1176

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1173#issuecomment-573663886):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**